.goreleaser.yaml

```diff
@@ -0,0 +1,45 @@
+# Goreleaser documentation at https://goreleaser.com
+before:
+  hooks:
+    # You may remove this if you don't use go modules.
+    - go mod tidy
+builds:
+  -
+    id: scalingo
+    binary: scalingo
+    main: ./scalingo
+    goos:
+      - linux
+      - windows
+      - darwin
+      - freebsd
+      - openbsd
+    goarch:
+      - amd64
+      - arm64
+      - 386
+    ignore:
+      - goos: darwin
+        goarch: 386
+
+    # Custom ldflags templates.
+    # https://goreleaser.com/customization/templates/
+    ldflags:
+      - -X main.version={{.Version}}
+        -X main.buildstamp={{.Date}}
+        -X main.githash={{.FullCommit}}
+
+archives:
+  -
+    name_template: 'scalingo_{{ .Version }}_{{ .Os }}_{{ .Arch }}{{ with .Arm }}v{{ . }}{{ end }}{{ with .Mips }}_{{ . }}{{ end }}{{ if not (eq .Amd64 "v1") }}{{ .Amd64 }}{{ end }}'
+    wrap_in_directory: true
+checksum:
+  name_template: 'checksums.txt'
+snapshot:
+  name_template: "{{ incpatch .Version }}-next"
```

Comment:
After reading the doc about snapshot (https://goreleaser.com/customization/snapshots), I'm not sure I understand why we define a snapshot generation here